doc/tutorial/statistical_inference/model_selection.rst

```diff
@@ -70,7 +70,7 @@ This example shows an example usage of the ``split`` method.
 
 The cross-validation can then be performed easily::
 
-    >>> kfold = KFold(n_splits=3)
+    >>> k_fold = KFold(n_splits=3)
```

Comment:
Good catch. Actually I am wondering whether this line should not be removed since `k_fold` is already defined a few lines above.